src/main/java/com/google/plantasy/utils/GameUtils.java

```diff
@@ -54,20 +54,13 @@ public static Entity createGameEntity(String gameName, DatastoreService datastor
     game.setQuizQuestion(quizQuestion);
     game.setQuizTimestamp(quiz_timestamp);
 
-    datastore.put(gameEntity);
+    datastore.put(game.getGameEntity());
```

Comment:
We can remove this put as it's the same as the put on line 61

Reply:
Without this put I get an incomplete key error; I don't think the entity key is created until it is placed in datastore.

Reply:
okay okay I see. 

So one quick thing, maybe move line 50 to be closer to line 59 because that's the first place we use that value. 

The other thing is... do we even need the gameId field? If the gameId == gameEntity.getKey then we can always get the game id by getting the key. That way, we don't have a redundant field in the datastore. 

We can still have a method in our Game wrapper class called getGameId() but it'll just return the keyToString(entity.getKey())

Reply:
Sounds good, this is out of scope for this pr but I'll follow up with another pr editing getGameId()